templates/contracts/ts/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "near-sdk-js build src/contract.ts build/hello_near.wasm",
     "deploy": "near dev-deploy --wasmFile build/hello_near.wasm",
-    "test": "cd sandbox-ts && $npm_execpath run test -- -- ../build/hello_near.wasm",
+    "test": "npm run build && cd sandbox-ts && $npm_execpath run test -- -- ../build/hello_near.wasm",
```

Comment:
lets use `$npm_execpath` so the user can choose their own package manager (e.g. npm, yarn, pnpm, etc)